### PR TITLE
Rewrite the blog article

### PR DESCRIPTION
**Pull request by nicojs**
A full rewrite of the blog article. 

It had all the content in it, but it wasn't a coherent article INMHO. I've rewritten it and removed some confusing things. It still needs some love (especially the "Start using Stryker.NET" part), but I would need to talk to @richardwerkman or @Mobrockers for that.

### CODE REVIEW

File: src/blog/2019-03-15/announcing-dotnet-framework-support.pug [hunk 1]
```diff
@@ -1,100 +1,135 @@
 extends ../../layout/blog-layout.pug
 block content
     :markdown-it
-        
-        ## Stryker.NET version 0.10
-        For people who have been following Stryker.NET over the last few months this change might not be a surprise. In version 0.10 Stryker.NET welcomes .NET Framework. For both .NET Core and .NET Framework Stryker.NET has a set of features available.
 
-        For them who don't know Stryker.NET yet, here are some feature highlights:
+        Stryker.NET has been under very active development. Up to now, we focussed on performance and .NET core.
+        That all changes with v0.10, as we now also support _.NET Framework_. Both framework are supported now,
```
Comment (ptoonen) [inline src/blog/2019-03-15/announcing-dotnet-framework-support.pug]: framework**s**

File: src/blog/2019-03-15/announcing-dotnet-framework-support.pug [hunk 2]
```diff
@@ -1,100 +1,135 @@
...
-    :markdown-it
-
-        #### Multi-platform
-        You can run Stryker.NET on Windows, Mac and Linux since it runs on .NET Core.
+        For those of you who might not be familiar with Stryker.NET and Mutation Testing, here is a brief reminder.
+        Please see [our RoboBar example](/example) for the full introduction.
+        
+        Stryker.NET places _mutants_ (bugs) inside your source code. After that, it runs your tests for each mutant.
+        If at least one tests fails, that's great! Your tests just killed that mutant. If not, too bad, the mutant survived.
+        apparently your missing a test.
```
Comment (ptoonen) [inline src/blog/2019-03-15/announcing-dotnet-framework-support.pug]: you're or you are

File: src/blog/2019-03-15/announcing-dotnet-framework-support.pug [hunk 3]
```diff
@@ -1,100 +1,135 @@
...
+    img(alt="stryker", src="/images/stryker-net/html-report-net.png" class="img-fluid")
 
-        On .NET Framework projects the solution file is required. You can provide this using:
-        ```
-        dotnet stryker --solution-path "../MySolution.sln"
-        ```
+    :markdown-it
 
-        If stryker is installed as a dotnet core 3.0 local tool, use the following instead:
+        It uses te generic [mutation-testing-elements](https://github.com/stryker-mutator/mutation-testing-elements),
```
Comment (ptoonen) [inline src/blog/2019-03-15/announcing-dotnet-framework-support.pug]: t**h**e

File: src/blog/2019-03-15/announcing-dotnet-framework-support.pug (hunk 2 above)
Comment (nicojs) [reply src/blog/2019-03-15/announcing-dotnet-framework-support.pug]: Thanks a lot, fixed it.

File: src/blog/2019-03-15/announcing-dotnet-framework-support.pug (hunk 1 above)
Comment (nicojs) [reply src/blog/2019-03-15/announcing-dotnet-framework-support.pug]: Thanks a lot, fixed it.

File: src/blog/2019-03-15/announcing-dotnet-framework-support.pug [hunk 4]
```diff
@@ -1,100 +1,135 @@
...
+        Because Stryker.NET supports VSTest, this performance boost wil work for most test frameworks, including [XUnit](https://xunit.github.io/),
+        [NUnit](https://nunit.org/) and [MSTest](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-mstest).
 
-    p *Local install needs .NET Core runtime 3.0+ to be installed
-    :markdown-it
-        ### 2. Install dependencies
-        For .NET Core projects this step can be skipped.
-        
-        For .NET Framework projects, make sure [nuget.exe](https://docs.microsoft.com/en-us/nuget/install-nuget-client-tools#windows) is installed on your system. Please follow their installation instructions.
+        Special thanks to [@_dupdob_](https://twitter.com/_dupdob_) for all his hard work on this feature!
```
Comment (Mobrockers) [inline src/blog/2019-03-15/announcing-dotnet-framework-support.pug]: While dupdob is a very active contributor and we appreciate his help very much, he did not actually contribute to the vstest integration so it's a bit weird to thank him for that :)

What he is working on is coverage analysis, which is mentioned further down in this article.

Comment (nicojs) [reply src/blog/2019-03-15/announcing-dotnet-framework-support.pug]: Ok, removed his name 👍

**Pull request merged**